docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -65,7 +65,7 @@
     "    import cirq\n",
```

Comment:
<p>Maybe instead of "accounts for single-qubit errors," say that it subtracts off single-qubit errors to isolate the error coming just from the two-qubit gate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>